string expressionName
float32 intensity
bool face_expression
